test/README.md

```diff
@@ -0,0 +1,15 @@
+## Usage
+
+### Prerequisites
+```bash
+pip install pytest
+```
+after changes in pyproject.toml do:
+```bash
+pip install -e .
```

Comment:
This should not be in the test folder

Reply:
Agree, either Let's add something like cd .. before and cd test afterwards or just shift the content to the Readme.md one level up.